packages/react-divider/src/components/Divider/Divider.types.ts

```diff
@@ -1,6 +1,24 @@
 import * as React from 'react';
 import { ComponentProps, ShorthandProps } from '@fluentui/react-utilities';
 
+/**
+ * Overrides react declaration of CSSProperties to allow using --divider-* as a valid style
+ */
+declare module 'react' {
+  export interface CSSProperties {
+    '--divider-borderMargin'?: string | number;
+    '--divider-flexDirection'?: string;
+    '--divider-fontColor'?: string;
+    '--divider-fontFamily'?: string;
+    '--divider-fontSize'?: string | number;
+    '--divider-fontWeight'?: string | number;
+    '--divider-lineHeight'?: string | number;
+    '--divider-borderSize'?: string | number;
```

Comment:
why allow the option to pass numbers ? will they represent px by default ?

Reply:
It could be confusing to projects (Teams) that by default use `rem`

Reply:
I'll remove then and require string!